x-pack/plugin/apm-data/src/main/resources/index-templates/logs-apm.error@template.yaml

```diff
@@ -26,6 +26,9 @@ template:
       processor.event:
         type: constant_keyword
         value: error
+      log.level:
+        type: constant_keyword
```

Comment:
I'm not sure about using `constant_keyword` for this. I think [OTel exception logs](https://opentelemetry.io/docs/specs/semconv/exceptions/exceptions-logs/) could explicitly set a log level to something other than "error", which could conflict with this.

Reply:
oh... I wasn't aware of that case. Should it then be mapped in `apm-data` and just added as a keyword?

Reply:
I'm not convinced we should set a default value for `log.level`, so wouldn't make a change to apm-data; but it makes sense to map the field as `keyword`.